### PR TITLE
Infinispan remote cache container testsuite

### PR DESCRIPTION
**Pull request by spriadka**

### CODE REVIEW

**Comment by hpehl** [comment]
Thanks, @spriadka for taking care of the new Infinispan subsystem.

File: tests/basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/remote/cache/container/ConnectionPoolTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    @AfterClass
+    public static void tearDown() throws IOException, CommandFailedException, OperationException {
+        operations.removeIfExists(remoteCacheContainerAddress(REMOTE_CACHE_CONTAINER_TO_BE_TESTED));
+        client.apply(new RemoveSocketBinding(REMOTE_SOCKET_BINDING));
+        client.apply(new RemoveSocketBinding(REMOTE_SOCKET_BINDING_CLUSTER));
+    }
+
+    @Drone
+    private WebDriver browser;
```
Comment (zebrik) [inline tests/basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/remote/cache/container/ConnectionPoolTest.java]: What about moving `browser`, `console`, `page` and `crudOperations` to parent `AbstractRemoteCacheContainerTest`? They are used by almost all children tests.

**Comment by spriadka** [comment]
Thanks, @zebrik, the code was fixed according to your comments, shall I merge now?

**Comment by zebrik** [comment]
Yes, feel free to merge. It's upon you if you will squash a bit or not.

**Pull request merged**